include/bl_storage.h

```diff
@@ -281,6 +357,7 @@ NRFX_STATIC_INLINE int read_life_cycle_state(enum lcs *lcs)
 
 	return 0;
 }
+#else
```

Comment:
isn't that nonintentional `#else`

Reply:
I thought I removed it. Seems I missed it.